wpilib/wpilib/command/command.py

```diff
@@ -10,6 +10,7 @@
 from ..robotstate import RobotState
 from ..sendable import Sendable
 from ..timer import Timer
+from networktables.instance import NetworkTablesInstance
```

Comment:
I would rather you do `from networktables import NetworkTables`, and then use `NetworkTables.NotifyFlags`.. it reads better, and we will always have a global NetworkTables instance.